core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -479,19 +479,20 @@ public Status getStatus() {
   /**
    * @return the status of the startup consistency check and inconsistent paths if it is complete
    */
-  public StartupConsistencyCheck getStartupConsistencyCheck() {
+  public StartupConsistencyCheckResult getStartupConsistencyCheck() {
     if (!Configuration.getBoolean(PropertyKey.MASTER_STARTUP_CONSISTENCY_CHECK_ENABLED)) {
-      return new StartupConsistencyCheck(StartupConsistencyCheck.Status.DISABLED, null);
+      return new StartupConsistencyCheckResult(StartupConsistencyCheckResult.Status.DISABLED, null);
     }
     if (!mStartupConsistencyCheck.isDone()) {
-      return new StartupConsistencyCheck(StartupConsistencyCheck.Status.RUNNING, null);
+      return new StartupConsistencyCheckResult(StartupConsistencyCheckResult.Status.RUNNING, null);
```

Comment:
nit: we could make this line happier by having static constructors for each type of result, then we wouldn't have to pass meaningless nulls or write `StartupConsistencyCheckResult` twice